### ISSUE TITLE
An Error Has Occured - Invalid Transaction

### ISSUE THREAD

**Issue by matmurdock** [dOpensource/dsiprouter#72]
I don't remember the exact error, but none of the lists would load.  I saw the following message in the logs "Can't reconnect until invalid transaction is rolled back".  Restarting the dSIPRouter service fixed the issue.  Not sure how to troubleshoot what transaction was locking things up.  Just an FYI.

Running .50

Mat

**Comment by mackhendricks** [comment]
Hey Mat,

Do you know what you were doing?  Were you adding a Carrier, PBX or something else?  Lot of stuff changed with this release and any info would be helpful.

Thanks,

**Comment by matmurdock** [comment]
The last thing I did was adjust the strip value on either the pbx or carrier last night.  I didn't notice the issue until this morning when I logged in.

Mat

**Comment by mackhendricks** [comment]
Thanks...that helps to narrow down the issue.  We have a bug that's scheduled to be fixed in 0.51 that sounds like the same issue.  I will give you a patch for this bug when it's ready.  The issue is below:

https://github.com/dOpensource/dsiprouter/issues/47

**Comment by matmurdock** [comment]
It seems to happen overnight even when no changes have been made.  I logged into dSIPRouter today and it was stuck again.

**Comment by mackhendricks** [comment]
Okay...we figure this out and issue a patch ASAP

**Comment by matmurdock** [comment]
Sounds good.  I have a support ticket in.  Looking forward to working with you guys.

**Comment by mackhendricks** [comment]
Yes...I saw that...thanks!  I will make sure we handle that on Tuesday.

In regards to this issue, we changed the way we dealt with stale database connections.   So, that looks like the culprit.  We should have a patch out ASAP and we will explain how to apply as well
